### PR TITLE
Proxmox VE network configuration: multiple fixes

### PR DESCRIPTION
**Pull request by sbraz**
I split this into multiple commits to make it easier to read and explain the reasoning behind each of them.

### CODE REVIEW

**Comment by sbraz** [comment]
I'm done refactoring the doc. I have tested the configuration on an ADV and a SCALE server. @gary-ovh, can you please test the configuration yourself too? I can grant you access to my servers if that helps, they are both attached to the same vRack and each of them has an additional IP routed to them.
There is a rather large commit ("rework sections and descriptions") because I had many sentences to change and I didn't know how to split it properly.

File: pages/bare_metal_cloud/dedicated_servers/proxmox-network-HG-Scale/guide.en-gb.md [hunk 1]
```diff
@@ -370,19 +370,35 @@ systemctl restart networking.service
...
-iface lo inet loopback
-iface ens18 inet static
-        address 46.105.135.97/28
-        gateway 46.105.135.110
-```
+> [!tabs]
+> Debian (ifupdown)
+>> Contents of the `/etc/network/interfaces` file:
+>>
+>> ```bash
```
Comment (remicmacs) [inline pages/bare_metal_cloud/dedicated_servers/proxmox-network-HG-Scale/guide.en-gb.md:381]:
```text

File: pages/bare_metal_cloud/dedicated_servers/proxmox-network-HG-Scale/guide.en-gb.md [hunk 2]
```diff
@@ -324,75 +294,129 @@ To configure the first usable IP address, you must edit the network configuratio
...
+#### Client VM configuration example
 
-Content of the file `/etc/network/interfaces`:
+The VM must be attached to the `vmbr1` bridge.
+
+> [!tabs]
+> Debian (ifupdown)
+>> Contents of the `/etc/network/interfaces` file:
+>>
+>> ```bash
```
Comment (remicmacs) [inline pages/bare_metal_cloud/dedicated_servers/proxmox-network-HG-Scale/guide.en-gb.md]:
```suggestion
>> ```text
```
According to the previous modifications

**Review by remicmacs — APPROVED** [review]
LGTM but as previously said on this topic, such changes can only be really validated via a lab, I think

**Comment by sbraz** [comment]
> LGTM but as previously said on this topic, such changes can only be really validated via a lab, I think

I did test the configuration on an ADV Gen 3 server and a HGR/SCALE one. Also @gary-ovh told me he would test the configuration as well.

**Comment by gary-ovh** [comment]
Thanks @sbraz! I tested all configs and everything is working perfectly!

@Y0Coss / @remicmacs this is ready for your review/translations

File: pages/bare_metal_cloud/dedicated_servers/proxmox-network-HG-Scale/guide.fr-fr.md [hunk 3]
```diff
@@ -16,7 +18,7 @@ updated: 2024-10-18
 ## Prérequis
 
 - Disposer d'un [serveur dédié OVHcloud](/links/bare-metal/bare-metal)
```
Comment (sbraz) [inline pages/bare_metal_cloud/dedicated_servers/proxmox-network-HG-Scale/guide.fr-fr.md]: J'enlèverais "disposer" du premier élément de la liste, ou alors on le remet sur le second

File: pages/bare_metal_cloud/dedicated_servers/proxmox-network-HG-Scale/guide.fr-fr.md [hunk 4]
```diff
@@ -33,273 +35,243 @@ Avec cette configuration, les adresses Additional IP doivent être attachées à
...
+Les étapes ci-dessous décrivent successivement comment :
 
 - se connecter en SSH sur Proxmox ;
-- créer un agrégat (linux bond), uniquement pour les gammes High Grade & SCALE ;
-- créer un bridge ;
-- autoriser le forwarding ;
-- ajouter les routes.
+- créer une interface d'agrégation (*bond*) (uniquement pour les gammes High Grade & Scale) ;
+- créer une interface bridge connectée au *bond* ;
+- autoriser le transfert de paquets entre les interfaces ;
```
Comment (sbraz) [inline pages/bare_metal_cloud/dedicated_servers/proxmox-network-HG-Scale/guide.fr-fr.md]: - autoriser le routage de paquets entre les interfaces ;

File: pages/bare_metal_cloud/dedicated_servers/proxmox-network-HG-Scale/guide.fr-fr.md [hunk 5]
```diff
@@ -33,273 +35,243 @@ Avec cette configuration, les adresses Additional IP doivent être attachées à
...
+>>         address 192.168.0.1/24
+>>         bridge-ports bond0
+>>         bridge-stp off
+>>         bridge-fd 0
+>>         # Ajoutez une seule Additional IP
+>>         up ip route add ADDITIONAL_IP/32 dev $IFACE
+>>         # Ajoutez un bloc IP
+>>         up ip route add ADDITIONAL_IP_BLOCK/28 dev $IFACE
+>>
 >> # Private
```
Comment (sbraz) [inline pages/bare_metal_cloud/dedicated_servers/proxmox-network-HG-Scale/guide.fr-fr.md]: Est-ce qu'on traduit Private ?

**Review by Y0Coss — APPROVED** [review]

File: pages/bare_metal_cloud/dedicated_servers/proxmox-network-HG-Scale/guide.fr-fr.md (hunk 5 above)
Comment (Y0Coss) [reply pages/bare_metal_cloud/dedicated_servers/proxmox-network-HG-Scale/guide.fr-fr.md]: Traduit et adapté aussi en EN

File: pages/bare_metal_cloud/dedicated_servers/proxmox-network-HG-Scale/guide.en-gb.md [hunk 6]
```diff
@@ -350,13 +350,13 @@ ssh PUB_IP_DEDICATED_SERVER
 >> auto lo
 >> iface lo inet loopback
 >>
->> # Public
+>> # Public interfaces
```
Comment (sbraz) [inline pages/bare_metal_cloud/dedicated_servers/proxmox-network-HG-Scale/guide.en-gb.md]: Attention dans ADV il n'y a qu'une interface publique et une privée donc pas de pluriel

**Comment by Y0Coss** [comment]
Thank you for your contribution @sbraz

**Pull request merged**